esmvaltool/cmorizers/data/formatters/datasets/esacci_seaice.py

```diff
@@ -0,0 +1,319 @@
+                    date_available = True
+                    break
+            if date_available:
+                full_list.append(new_list[idx])
+            else:
+                logger.debug("No data available for %d/%d", loop_date.month,
+                             loop_date.year)
+                nan_cube = _create_nan_cube(new_list[0], loop_date.year,
+                                            loop_date.month, loop_date.day,
+                                            is_daily)
+                full_list.append(nan_cube)
+            loop_date += relativedelta.relativedelta(months=1)
+
+    iris.util.unify_time_units(full_list)
+    cube = full_list.concatenate_cube()
+    cube.coord('time').points = cube.coord('time').core_points().astype(
+        'float64')
+
+    # Set correct names
+    cube.var_name = definition.short_name
+    cube.standard_name = definition.standard_name
+    cube.long_name = definition.long_name
+
+    # Fix units
+    cube.units = definition.units
+
+    # Fix ocean-type grid (2-dim lat + lon)
+    fixcube = OceanFixGrid(definition)
+    cube = fixcube.fix_metadata(cubes=[cube])[0]
+
+    # Fix coordinates
+    cube = _fix_coordinates(cube, definition)
+    cube.coord('latitude').attributes = None
+    cube.coord('longitude').attributes = None
+
+    # add aux coord 'typesi'
+    area_type = AuxCoord([1.0], standard_name='area_type', var_name='type',
+                         long_name='Sea Ice area type')
+    cube.add_aux_coord(area_type)
+
+    # add attribute cell_measures
+#    siconc:cell_measures = "area: areacello"
+#    cube.attributes.update({"cell_meaures": "area: areacello"})
+    cube.attributes.locals['cell_measures'] = 'area: areacello'
+
+    # Fix data type
+    cube.data = cube.core_data().astype('float32')
+
+    # save daily results
+    logger.debug("Saving cube\n%s", cube)
+    logger.debug("Setting time dimension to UNLIMITED while saving!")
+    version = attributes['version']
+    attributes['version'] = f'{version}-{region}'
+    save_variable(cube, cube.var_name,
+                  out_dir, attributes,
+                  unlimited_dimensions=['time'])
+
+    # calculate monthly means
+    cube = monthly_statistics(cube, operator='mean')
+    # Remove monthly statistics aux coordinates
+    cube.remove_coord(cube.coord('month_number'))
+    cube.remove_coord(cube.coord('year'))
+    # save monthly results
+    logger.debug("Saving cube\n%s", cube)
+    logger.debug("Setting time dimension to UNLIMITED while saving!")
+    version = attributes['version']
+    attributes['mip'] = var['mip2']
+    definition = cmor_table.get_variable(var['mip2'], var['short_name'])
+    save_variable(cube, cube.var_name,
+                  out_dir, attributes,
+                  unlimited_dimensions=['time'])
+
+    # create and save areacello
+    # (code adadapted from formatter 'nsidc_g02202_sh.py')
+    _create_areacello(cfg, cube, attributes, out_dir)
```

Comment:
It might be good to have the option to not create areacello. I ran the CMORiser using a series of PBS jobs, one per year to speed-up the process and I ended up having 20+ areacello.